docs/_templates/layout.html

```diff
@@ -84,3 +84,19 @@
         docSearchReady();
     </script>
 {% endblock %}
+
+{% block footer %}
+    {{ super() }}
+    <template data-promotion-banner data-date-until="2022-06-18">
```

Comment:
note - date was listed using 5 for the month in JS but that was because of JS date parsing when given numbers, this date format can now just be iso date  which is a bit easier to understand.